fuse/ipns/ipns_test.go

```diff
@@ -131,9 +189,45 @@ func TestFilePersistence(t *testing.T) {
 	}
 }
 
+func TestDeeperDirs(t *testing.T) {
```

Comment:
Maybe I should rename it. I just wanted to test the case where  the parent of a directory was a directory.